app/helpers/script_helper.rb

```diff
@@ -1,17 +1,19 @@
-require 'set'
-
 # rubocop:disable Rails/HelperInstanceVariable
 module ScriptHelper
   include Webpacker::Helper
 
-  def javascript_pack_tag_once(name)
-    @scripts ||= Set.new
-    @scripts.add(name)
+  def javascript_packs_tag_once(*names, prepend: false)
+    @scripts ||= []
+    if (prepend)
+      @scripts = names | @scripts
+    else
+      @scripts = @scripts | names
```

Comment:
We can write this as:

```suggestion
      @scripts |= names
```